src/sphinx_book_theme/__init__.py

```diff
@@ -117,7 +131,7 @@ def hash_html_assets(app, pagename, templatename, context, doctree):
     # run but the book theme CSS file won't be linked in Sphinx.
     if app.config.html_theme == "sphinx_book_theme":
         assets.append("styles/sphinx-book-theme.css")
-    hash_assets_for_files(assets, get_html_theme_path() / "static", context)
+    hash_assets_for_files(assets, get_html_theme_path() / "static", context, app)
```

Comment:
This function is public, but I can't imagine anyone using it. Are you happy with the breaking change?